gaia_spatialstats/processes.py

```diff
@@ -18,6 +18,8 @@
 ###############################################################################
 import logging
 import numpy as np
+import pandas as pd
+import json
```

Comment:
to keep in sync with pep8 style, should we have a line between groups of imports?